Ports/git/package.sh

```diff
@@ -9,7 +9,6 @@ configopts=(
     "--target=${SERENITY_ARCH}-pc-serenity"
     "--with-lib=${SERENITY_INSTALL_ROOT}/usr/local"
```

Comment:
I presume this could be dropped as well, but would need to test the build to be sure

https://github.com/git/git/blob/406f326d271e0bacecdb00425422c5fa3f314930/configure.ac#L171-L189

Reply:
the target likely has to stay; `--with-lib` could probably be removed, but i'd say it's out of the scope of this MR; I'll be going through *all* of the packages and cleaning them up + doing upgrades (if spoons allow)